src/components/molecules/CallOutMessageForm/CallOutMessageForm.tsx

```diff
@@ -10,6 +10,7 @@ interface PropsType {
   isMessageToTheBandSent: boolean;
 }
 
+// @debt This file doesn't appear to be used anywhere anymore. We can probably remove it in our next tech-debt cleanup pass
```

Comment:
you can also check 
`src/components/molecules/UserSearchBar/...`
seems the files in the directory are also not used anywhere.

Reply:
@sunny-viktoryia Commented on your PR, probably makes sense for you to remove it there.

Reply:
the files are removed: https://github.com/sparkletown/sparkle/commit/2574d407862199372cf770036bc25c76283e698d

Reply:
Cool, thanks :)